zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -4915,6 +4987,28 @@ components:
         signalKey:
           description: The unique ID of the signal that was broadcast.
           type: string
+    ResourceResult:
+      type: object
+      properties:
+        resourceName:
+          description: Resource name of this resource.
+          type: string
+        version:
+          description: Version of this resource.
```

🔧 Let's align to other endpoints:

```suggestion
          description: The assigned resource version.
```